packages/wdio-config/src/lib/ConfigParser.js

```diff
@@ -214,8 +214,17 @@ export default class ConfigParser {
         const filesToFilter = new Set()
         const fileList = ConfigParser.getFilePaths(config)
         cliArgFileList.forEach(filteredFile => {
+            let matchedFiles = ConfigParser.getFilePaths(glob.sync(filteredFile))
+            let globMatchedFiles = []
+            matchedFiles.forEach(file => {
+                if (matchedFiles.indexOf(file) != -1) {
```

Comment:
I don't get this statement. You're verifying that the array you're looping has the value that you got from that same array?

Reply:
@baruchvlz - it was left over from a previous refactoring.  You're right, it doesn't make sense.

Let me remove that section & clean it up.